Doc/library/http.server.rst

```diff
@@ -51,9 +51,42 @@ handler.  Code to create and run the server looks like this::
    .. versionadded:: 3.7
 
 
-The :class:`HTTPServer` and :class:`ThreadingHTTPServer` must be given
-a *RequestHandlerClass* on instantiation, of which this module
-provides three different variants:
+.. class:: HTTPSServer(server_address, RequestHandlerClass, \
+                       bind_and_activate=True, *, certfile, keyfile=None, \
+                       password=None, alpn_protocols=None)
+
+   This class is a :class:`HTTPServer` subclass with a wrapped socket using the
+   :mod:`ssl`, if the :mod:`ssl` module is not available the class will not
+   initialize.
+
+   The *certfile* argument is required and is the path to the SSL
+   certificate chain file. The *keyfile* is the path to its private key. But
+   private keys are often protected and wrapped with PKCS #8, so we provide
+   *password* argument for that case.
```

Comment:
```suggestion
   The *certfile* argument is the path to the SSL certificate chain file,
   and the *keyfile* is the path to file containing the private key.
   
   A *password* can be specified for files protected and wrapped with PKCS#8,
   but beware that this could possibly expose hardcoded passwords in clear.
```